File: bruteforce/1436.c
#include <stdio.h>

int main(){
    int n, d;
    int temp = 0;
    int cnt = 1;
    int num = 1666;
    scanf("%d", &n);

    if (n==1)
        printf("666");
    else{
        while (1){
            d = num;
            while (d > 0){
                if (d%10 == 6)
                    temp ++;
                else
                    temp=0;
                
                if (temp == 3)
                    break;

                d = d / 10;
            }
            
            if (temp == 3){
                cnt++;
                if (cnt == n){
                    printf("%d", num);
                    break;
                }
            }

            num++;
            temp=0;
        }
    }
    return 0;
}

File: bruteforce/1436.py
n = int(input())
a = [666, 1666, 2666, 3666, 4666, 5666]
num = 6660
cnt = 6

if n >= 1 and n <= 6:
    print(a[n-1])
else:
    while True:
        if '666' in str(num):
            cnt += 1
            if n == cnt:
                print(num)
                break

        num += 1
